root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -746,7 +746,7 @@ limited_link_type_combinations: [
              input_entity_type: 'recording',
     expected_relationship_type: 'streamingfree',
             expected_clean_url: 'https://audiomack.com/dablixx-osha/song/they-cant-understand',
-       only_valid_entity_types: ['recording'],
+       only_valid_entity_types: ['recording', 'release'],
```

Comment:
Would having a test with `input_entity_type: 'release'` be useful at all, or is it redundant if it's already contained in `only_valid_entity_types`?

Reply:
AFAIU it shouldn't matter since we already know both are valid.